libs/langchain/langchain/chains/qa_with_sources/base.py

```diff
@@ -120,9 +120,9 @@ def validate_naming(cls, values: Dict) -> Dict:
 
     def _split_sources(self, answer: str) -> Tuple[str, str]:
         """Split sources from answer."""
-        if re.search(r"SOURCES?[:\s]", answer, re.IGNORECASE):
+        if re.search(r"SOURCES?[:]\s", answer, re.IGNORECASE):
```

Comment:
are the square brackets still needed if there's only one element?

also what's the failure mode here? is it that "sources: " would be split before the ending space? does that matter given we strip the source?

Reply:
Hey @baskaryan  thanks for the response. You're right I don't need the square braces.
The failure was the fact that if we had the word "source" or "sources" in our answer it would cut the answer off. 
Example:
The source of truth for the test subject. SOURCES: 28-pl.
Current Answer:
The
Expected Answer:
The source of truth for the test subject. 
SOURCES: 28pl

Reply:
I think we can also do away with the whitespace regex, since we perform a strip. Let me update my PR.

Reply:
just fixed it!